UI/js-src/lsmb/menus/Cache.js

```diff
@@ -0,0 +1,41 @@
+define(["dojo/_base/lang","dojo/when" /*=====, "dojo/_base/declare", "dojo/store/api/Store" =====*/],

```

Comment:
But then, why not retrieve the whole tree at once? For most sessions the tree won't change during the session.